openjdk-11.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: openjdk-11
   version: 11.0.17.8
-  epoch: 0
+  epoch: 1
```

Comment:
Is this correct to bump the epoch as the package will include extra runtime dependencies.

Reply:
Yes.